gitextensions/gitextensions
# Fix #3520. Implemented feature request

Fixes #3520 

### Changes proposed in this pull request:

- Add new settings that in fact duplicates and stores values from main plugin window. Now default values for "Delete obsoleted branches" plugin fields pulled from settings.
- By the way. Remove redundant handlers (ClearResults method wrappers for specific control).
 
### Screenshots before and after (if PR changes UI):

#### Before
![delete_obsoleted_branches_before](https://user-images.githubusercontent.com/29478527/35572804-7bacfb1a-05e6-11e8-8f09-6daf7e8ee695.png)

#### After
![delete_obsoleted_branches_after](https://user-images.githubusercontent.com/29478527/35572803-7b1eec30-05e6-11e8-9937-c3d651b42b24.png)

#### What did I do to test the code and ensure quality:
 - Run GitExtensions
 - Open any repository
 - Goto plugins -> Settings ->"Delete obsoleted branches" plugin
 - check that "Delete obsoleted branches" settings are new

#### Has been tested on (remove any that don't apply):
 - GIT 2.14 and above
 - Windows 10
